status-im/status-desktop
# feat(connector)_: use status image for DApps connected with Status connector

### Description

Use Status connector badge for DApps connection with Status Connector. Previously only Wallet connect badge was displayed

### What does the PR do

- Added a new image for status connector
- Update the Modal to accept a new parameter
- Keep the change retro-compatible by making parameter not required

### Affected areas

- Request to connect account
- Request for signing transaction

### Architecture compliance

My PR is consistent with this document: [Architecture guidelines](../architecture-guide.md)

### Screenshot of functionality (including design for comparison)


[Screencast from 2024-08-13 15-35-43.webm](https://github.com/user-attachments/assets/d1bebbe8-b832-471a-805b-f2c840e953e7)

### Impact on end user

User will now see a status icon instead of Wallet connect icon for Dapps connection with Status connector

### How to test

- Launch one instance of Status desktop
- Use a DApps interface to connect with Wallet connect and make sure the icon is the Wallet connect one
- Use a Dapps interface to connect with Status and make sure the icon is the Status connector one

### Risk 

No specific risks

Tick **one**:
